### ISSUE TITLE
ERROR: Unable to parse yarn audit output

### ISSUE THREAD

**Issue by RaSeR** [djfdyuruiry/improved-yarn-audit#5]
```
yarn run improved-yarn-audit --min-severity high
yarn run v1.22.0
$ /~~~~~/node_modules/.bin/improved-yarn-audit --min-severity high
Minimum severity level to report: high

Running yarn audit...

ERROR: Unable to parse yarn audit output, raw output:
[{"type":"auditAdvisory","data":{"resolution":{"id":786,"path":"babel-jest>babel-plugin-istanbul>test-exclude>micromatch>braces","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"1.8.5","paths":["babel-jest>babel-plugin-istanbul>test-exclude>micromatch>braces"]}],"id":786,"created":"2019-02-15T21:44:30.680Z","updated":"2019-04-02T18:18:29.356Z","deleted":null,"title":"Regular Expression Denial of Service","found_by":{"link":"","name":"Santosh Rao"},"reported_by":{"link":"","name":"Santosh Rao"},"module_name":"braces","cves":[],"vulnerable_versions":"<2.3.1","patched_versions":">=2.3.1","overview":"Versions of `braces` prior to 2.3.1 are vulnerable to Regular Expression Denial of Service (ReDoS). Untrusted input may cause catastrophic backtracking while matching regular expressions. This can cause the application to be unresponsive leading to Denial of Service.","recommendation":"Upgrade to version 2.3.1 or higher.","references":"- [GitHub Commit](https://github.com/micromatch/braces/commit/abdafb0cae1e0c00f184abbadc692f4eaa98f451)","access":"public","severity":"low","cwe":"CWE-185","metadata":{"module_type":"","exploitability":4,"affected_components":""},"url":"https://npmjs.com/advisories/786"}}},{"type":"auditAdvisory","data":{"resolution":{"id":1488,"path":"jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"6.3.0","paths":["@vue/cli-plugin-babel>webpack>acorn","@vue/cli-plugin-eslint>webpack>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest-environment-jsdom-fifteen>jsdom>acorn-globals>acorn","eslint>espree>acorn","eslint-plugin-vue>vue-eslint-parser>espree>acorn","webpack>acorn","webpack-stream>webpack>acorn"]},{"version":"5.7.3","paths":["@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn","jsdom>acorn"]},{"version":"6.3.0","paths":["@vue/cli-service>webpack>acorn"]},{"version":"6.3.0","paths":["@vue/cli-service>webpack-bundle-analyzer>acorn"]},{"version":"5.7.3","paths":["gulp-sourcemaps>@gulp-sourcemaps/identity-map>acorn"]},{"version":"5.7.3","paths":["gulp-sourcemaps>acorn"]}],"id":1488,"created":"2020-03-02T19:21:25.485Z","updated":"2020-03-10T17:44:45.038Z","deleted":null,"title":"Regular Expression Denial of Service","found_by":{"link":"","name":"Peter van der Zee","email":""},"reported_by":{"link":"","name":"Peter van der Zee","email":""},"module_name":"acorn","cves":[],"vulnerable_versions":">=5.5.0 <5.7.4 || >=6.0.0 <6.4.1 || >=7.0.0 <7.1.1","patched_versions":">=5.7.4 <6.0.0 || >=6.4.1 <7.0.0 || >=7.1.1","overview":"Affected versions of `acorn` are vulnerable to Regular Expression Denial of Service. A regex in the form of `/[x-\\ud800]/u` causes the parser to enter an infinite loop. The string is not valid UTF16 which usually results in it being sanitized before reaching the parser. If an application processes untrusted input and passes it directly to `acorn`, attackers may leverage the vulnerability leading to Denial of Service.","recommendation":"Upgrade to versions 5.7.4, 6.4.1, 7.1.1 or later.","references":"","access":"public","severity":"moderate","cwe":"CWE-400","metadata":{"module_type":"","exploitability":3,"affected_components":""},"url":"https://npmjs.com/advisories/1488"}}},{"type":"auditAdvisory","data":{"resolution":{"id":1488,"path":"jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"6.3.0","paths":["@vue/cli-plugin-babel>webpack>acorn","@vue/cli-plugin-eslint>webpack>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn-globals>acorn","jsdom>acorn-globals>acorn","@vue/cli-plugin-unit-jest>jest-environment-jsdom-fifteen>jsdom>acorn-globals>acorn","eslint>espree>acorn","eslint-plugin-vue>vue-eslint-parser>espree>acorn","webpack>acorn","webpack-stream>webpack>acorn"]},{"version":"5.7.3","paths":["@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","@vue/cli-plugin-unit-jest>jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn","jest>jest-cli>@jest/core>jest,<--- Last few GCs --->,,[9239:0x103b13000]    38315 ms: Mark-sweep 1396.8 (1455.7) -> 1396.8 (1455.7) MB, 80.8 / 0.0 ms  (average mu = 0.145, current mu = 0.006) allocation failure scavenge might not succeed,[9239:0x103b13000]    38391 ms: Mark-sweep 1396.8 (1455.7) -> 1396.8 (1455.7) MB, 75.7 / 0.0 ms  (average mu = 0.080, current mu = 0.002) allocation failure scavenge might not succeed,,,<--- JS stacktrace --->,,==== JS stack trace =========================================,,Security context: 0x204a8f21e6e9 <JSObject>,    0: builtin exit frame: stringify(this=0x204a66933ea9 <Object map = 0x204a05c8b9f1>,0x204aab6026f1 <undefined>,0x204aab6026f1 <undefined>,0x204a914371f9 <Object map = 0x204a05ce8e81>,0x204a66933ea9 <Object map = 0x204a05c8b9f1>),,    1: arguments adaptor frame: 1->3,    2: stringify [0x204ab655d591] [/usr/local/Cellar/yarn/1.22.0/libexec/lib/cli.js:~104818] [pc=0x2e288b7fbb7c](this=0x204acdf0...,,FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory, 1: 0x10096dd2e node::Abort() (.cold.1) [/usr/local/Cellar/node@10/10.19.0/bin/node], 2: 0x10003b906 node_module_register [/usr/local/Cellar/node@10/10.19.0/bin/node], 3: 0x10003bac7 node::OnFatalError(char const*, char const*) [/usr/local/Cellar/node@10/10.19.0/bin/node], 4: 0x100178fd3 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/usr/local/Cellar/node@10/10.19.0/bin/node], 5: 0x100178f75 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/usr/local/Cellar/node@10/10.19.0/bin/node], 6: 0x10042d4e8 v8::internal::Heap::UpdateSurvivalStatistics(int) [/usr/local/Cellar/node@10/10.19.0/bin/node], 7: 0x10042f125 v8::internal::Heap::CheckIneffectiveMarkCompact(unsigned long, double) [/usr/local/Cellar/node@10/10.19.0/bin/node], 8: 0x10042c7c2 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/usr/local/Cellar/node@10/10.19.0/bin/node], 9: 0x10042b5d3 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/usr/local/Cellar/node@10/10.19.0/bin/node],10: 0x100433bfa v8::internal::Heap::AllocateRawWithLigthRetry(int, v8::internal::AllocationSpace, v8::internal::AllocationAlignment) [/usr/local/Cellar/node@10/10.19.0/bin/node],11: 0x100433c44 v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationSpace, v8::internal::AllocationAlignment) [/usr/local/Cellar/node@10/10.19.0/bin/node],12: 0x10041429b v8::internal::Factory::AllocateRawWithImmortalMap(int, v8::internal::PretenureFlag, v8::internal::Map*, v8::internal::AllocationAlignment) [/usr/local/Cellar/node@10/10.19.0/bin/node],13: 0x100415b43 v8::internal::Factory::NewRawOneByteString(int, v8::internal::PretenureFlag) [/usr/local/Cellar/node@10/10.19.0/bin/node],14: 0x100666d85 v8::internal::IncrementalStringBuilder::Extend() [/usr/local/Cellar/node@10/10.19.0/bin/node],15: 0x1004c38a7 v8::internal::JsonStringifier::SerializeString(v8::internal::Handle<v8::internal::String>) [/usr/local/Cellar/node@10/10.19.0/bin/node],16: 0x1004c4b92 v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<false>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],17: 0x1004c79b1 v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<true>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],18: 0x1004c5b74 v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<false>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],19: 0x1004c79b1 v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<true>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],20: 0x1004c7e8a v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<true>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],21: 0x1004c7e8a v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<true>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],22: 0x1004c5b74 v8::internal::JsonStringifier::Result v8::internal::JsonStringifier::Serialize_<false>(v8::internal::Handle<v8::internal::Object>, bool, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],23: 0x1004c22bd v8::internal::JsonStringifier::Stringify(v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>) [/usr/local/Cellar/node@10/10.19.0/bin/node],24: 0x100209915 v8::internal::Builtin_Impl_JsonStringify(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/usr/local/Cellar/node@10/10.19.0/bin/node],25: 0x2e288b7dbf9d ,26: 0x2e288b78a5c3 ,27: 0x2e288b7fbb7c ]
error Command failed with exit code 1.
```

Any else info needed?

**Comment by djfdyuruiry** [comment]
Hi @RaSeR, could I get a list of the dependencies you have in your `package.json`?

**Comment by RaSeR** [comment]
mostly like this:
```json
"dependencies": {
    "axios": "0.19.2",
    "core-js": "^3.6.4",
    "handlebars": "^4.7.3",
    "hooper": "^0.3.4",
    "intersection-observer": "0.7.0",
    "lodash": "^4.17.15",
    "moment": "2.24.0",
    "moment-timezone": "^0.5.28",
    "normalize.css": "8.0.1",
    "plural-ru": "^2.0.2",
    "storage-factory": "^0.1.0",
    "v-autocomplete": "1.8.2",
    "vue": "^2.6.11",
    "vue-bem-cn": "^1.1.1",
    "vue-modal-dialogs": "^3.0.0",
    "vue-text-mask": "^6.1.2",
    "vuex": "^3.1.3"
  },
  "devDependencies": {
    "@babel/cli": "^7.8.4",
    "@babel/core": "^7.8.7",
    "@babel/helpers": "^7.8.4",
    "@babel/plugin-proposal-optional-chaining": "^7.8.3",
    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
    "@babel/plugin-syntax-jsx": "^7.8.3",
    "@babel/plugin-transform-modules-commonjs": "^7.8.3",
    "@babel/plugin-transform-runtime": "^7.8.3",
    "@babel/polyfill": "^7.8.7",
    "@babel/preset-env": "^7.8.7",
    "@babel/runtime": "^7.8.7",
    "@docdoc/bundle-size": "^0.0.8",
    "@docdoc/test-coverage": "^1.0.1",
    "@intervolga/optimize-cssnano-plugin": "^1.0.6",
    "@vue/cli-service": "^4.2.3",
    "@vue/eslint-config-airbnb": "^5.0.2",
    "@vue/test-utils": "^1.0.0-beta.32",
    "autoprefixer": "^9.7.4",
    "babel-core": "7.0.0-bridge.0",
    "babel-eslint": "^10.1.0",
    "babel-helper-vue-jsx-merge-props": "2.0.3",
    "babel-jest": "^25.1.0",
    "babel-loader": "8.0.6",
    "babel-plugin-lodash": "^3.3.4",
    "babel-plugin-transform-vue-jsx": "4.0.1",
    "css-loader": "3.4.2",
    "cssnano": "4.1.10",
    "cssnano-preset-advanced": "4.0.7",
    "duplicate-package-checker-webpack-plugin": "^3.0.0",
    "eslint": "^6.8.0",
    "eslint-config-airbnb-base": "^14.1.0",
    "eslint-plugin-import": "^2.20.1",
    "eslint-plugin-vue": "^6.2.2",
    "file-loader": "6.0.0",
    "gulp": "4.0.2",
    "gulp-concat": "2.6.1",
    "gulp-if": "3.0.0",
    "gulp-load-plugins": "^2.0.2",
    "gulp-sourcemaps": "2.6.5",
    "gulp-uglify": "2.1.2",
    "hard-source-webpack-plugin": "^0.13.1",
    "html-loader": "0.5.5",
    "husky": "^4.2.3",
    "image-webpack-loader": "^6.0.0",
    "jest": "^25.1.0",
    "jest-css-modules": "^2.1.0",
    "jest-junit": "^10.0.0",
    "jest-localstorage-mock": "^2.4.0",
    "js-beautify": "1.10.3",
    "jsdom": "16.2.1",
    "less": "^3.11.1",
    "less-loader": "^5.0.0",
    "lint-staged": "^10.0.8",
    "lodash-webpack-plugin": "^0.11.5",
    "mini-css-extract-plugin": "^0.9.0",
    "moment-locales-webpack-plugin": "1.2.0",
    "moment-timezone-data-webpack-plugin": "^1.1.0",
    "mq-polyfill": "^1.1.8",
    "node-object-hash": "^2.0.0",
    "npm-run-all": "^4.1.5",
    "optimize-css-assets-webpack-plugin": "^5.0.3",
    "postcss": "^7.0.27",
    "postcss-import": "12.0.1",
    "postcss-inline-svg": "^4.1.0",
    "postcss-less": "^3.1.4",
    "postcss-loader": "3.0.0",
    "postcss-preset-env": "6.7.0",
    "postcss-pxtorem": "5.1.1",
    "postcss-size": "3.0.0",
    "prettier": "^1.19.1",
    "stream-combiner2": "1.1.1",
    "style-loader": "1.1.3",
    "stylelint": "^13.2.1",
    "stylelint-at-rule-no-children": "^0.3.1",
    "stylelint-config-prettier": "^8.0.1",
    "stylelint-config-recommended": "^3.0.0",
    "stylelint-declaration-block-no-ignored-properties": "^2.3.0",
    "stylelint-media-use-custom-media": "^1.0.0",
    "stylelint-order": "^4.0.0",
    "stylelint-prettier": "^1.1.2",
    "stylelint-value-no-unknown-custom-properties": "^2.0.0",
    "terser-webpack-plugin": "^2.3.5",
    "url-loader": "^4.0.0",
    "vinyl-named": "1.1.0",
    "vue-jest": "^3.0.5",
    "vue-loader": "15.9.0",
    "vue-svgicon": "^3.2.6",
    "vue-template-compiler": "^2.6.11",
    "webpack": "^4.42.0",
    "webpack-bundle-analyzer": "^3.6.1",
    "webpack-cli": "^3.3.11",
    "webpack-stream": "^5.2.1",
    "webpackbar": "^4.0.0"
  },
```

**Comment by joelbiffin** [comment]
This problem comes from the `JSON.parse()` used inside this package. It exceeds the maximum stack size. Could be fixed by using some kind of streaming JSON.parse

**Comment by djfdyuruiry** [comment]
@RaSeR I was unable to reproduce the issue using node 10 and v1.1.1 of the package. I have published a v1.2.0 of the package now with the changes suggested by @joelbiffin. Can you retest with this new version?

**Comment by RaSeR** [comment]
We moved from node v8 to node v10 and clean up some outdated packages. And for now there is no problem that I can reproduce(( So anyway thx for your time :)

**Comment by felixgourdeau** [comment]
I have the same issue on my project. `yarn audit --json > audit-output.json` generates a 1.5Gb files in my case, and JSON.parse() in improved-yarn-audit can't handle it.

using node 12.6.0

` "dependencies": {
    "@loadable/component": "^5.10.2",
    "@loadable/server": "^5.10.2",
    "@material-ui/core": "^4.9.8",
    "ajv": "^6.12.0",
    "aws-sdk": "^2.602.0",
    "axios": "^0.19.0",
    "body-parser": "^1.19.0",
    "clsx": "^1.0.4",
    "connect-redis": "^3.4.2",
    "dotenv": "^8.2.0",
    "ejs": "^2.6.2",
    "email-validator": "^2.0.4",
    "express": "^4.17.1",
    "express-async-errors": "^3.1.1",
    "express-rate-limit": "^5.1.1",
    "express-session": "^1.16.2",
    "helmet": "^3.21.2",
    "json-colorizer": "^2.2.1",
    "jsonschema": "^1.2.6",
    "jsonwebtoken": "^8.5.1",
    "jwk-to-pem": "^2.0.1",
    "lodash": "^4.17.15",
    "lru-cache": "^5.1.1",
    "newrelic": "^6.2.0",
    "node-fetch": "^2.6.0",
    "node-polyglot": "^2.3.1",
    "pfu-node-logger": "^1.4.8",
    "p-queue": "^6.3.0",
    "query-string": "^6.8.2",
    "ramda": "^0.26.1",
    "rate-limit-redis": "^1.7.0",
    "react": "16.9.0",
    "react-dom": "16.9.0",
    "react-redux": "^7.1.0",
    "react-router": "5.0.1",
    "react-router-dom": "5.0.1",
    "redis": "^2.8.0",
    "redux": "^4.0.4",
    "redux-saga": "^1.0.5",
    "reselect": "^4.0.0",
    "serialize-javascript": "3.0.0",
    "source-map-support": "^0.5.16",
    "typesafe-actions": "^4.4.2",
    "uuid": "^3.3.2"
  },
  "devDependencies": {
    "0x": "^4.9.1",
    "@babel/core": "^7.5.4",
    "@babel/plugin-proposal-class-properties": "^7.5.0",
    "@babel/plugin-proposal-optional-chaining": "^7.8.3",
    "@babel/plugin-transform-runtime": "^7.5.0",
    "@babel/preset-env": "^7.5.4",
    "@babel/preset-react": "^7.0.0",
    "@babel/preset-typescript": "^7.3.3",
    "@cypress/browserify-preprocessor": "^2.1.1",
    "@cypress/webpack-preprocessor": "^4.1.0",
    "@loadable/babel-plugin": "^5.10.0",
    "@loadable/webpack-plugin": "^5.7.1",
    "@types/axios": "^0.14.0",
    "@types/connect-redis": "^0.0.10",
    "@types/cypress-cucumber-preprocessor": "^1.12.0",
    "@types/ejs": "^2.6.3",
    "@types/express": "^4.17.0",
    "@types/express-rate-limit": "^5.0.0",
    "@types/express-session": "^1.15.13",
    "@types/helmet": "^0.0.45",
    "@types/jest": "^24.0.18",
    "@types/jsonwebtoken": "^8.3.2",
    "@types/loadable__component": "^5.10.0",
    "@types/loadable__server": "^5.9.1",
    "@types/lodash": "^4.14.139",
    "@types/lru-cache": "^5.1.0",
    "@types/node-fetch": "^2.5.0",
    "@types/node-polyglot": "^0.4.33",
    "@types/ramda": "^0.26.16",
    "@types/rate-limit-redis": "^1.6.1",
    "@types/react": "^16.9.27",
    "@types/react-dom": "^16.8.4",
    "@types/react-redux": "^7.1.1",
    "@types/react-router": "^5.0.3",
    "@types/react-router-dom": "^4.3.4",
    "@types/redis": "^2.8.14",
    "@types/redux": "^3.6.0",
    "@types/serialize-javascript": "^1.5.0",
    "@types/uuid": "^3.4.5",
    "@typescript-eslint/eslint-plugin": "2.19.2",
    "@typescript-eslint/parser": "2.19.2",
    "autocannon": "^4.4.1",
    "autoprefixer": "^9.6.1",
    "babel-loader": "^8.0.6",
    "concurrently": "^5.0.0",
    "copy-webpack-plugin": "^5.0.3",
    "cross-env": "^6.0.3",
    "css-loader": "^3.1.0",
    "cypress": "3.4.1",
    "cypress-cucumber-preprocessor": "1.16.0",
    "eslint": "^6.0.1",
    "eslint-config-prettier": "^6.0.0",
    "eslint-loader": "^2.2.1",
    "eslint-plugin-prettier": "^3.1.0",
    "eslint-plugin-react": "^7.14.2",
    "favicons-webpack-plugin": "3.0.1",
    "file-loader": "^4.1.0",
    "fork-ts-checker-webpack-plugin": "^1.4.3",
    "http-server": "0.12.1",
    "improved-yarn-audit": "^2.0.0",
    "jest": "25.1.0",
    "jest-html-reporter": "^2.8.0",
    "jest-multiple-result-processors": "^0.0.3",
    "jest-sonar-reporter": "^2.0.0",
    "lighthouse-ci": "^1.10.0",
    "mini-css-extract-plugin": "^0.8.0",
    "nodemon": "^1.19.1",
    "postcss-flexbugs-fixes": "^4.1.0",
    "postcss-import": "^12.0.1",
    "postcss-loader": "^3.0.0",
    "postcss-mixins": "^6.2.2",
    "prettier": "1.19.1",
    "redis-mock": "^0.49.0",
    "redux-devtools-extension": "^2.13.8",
    "rimraf": "^2.6.3",
    "sonarqube-scanner": "^2.5.0",
    "terser-webpack-plugin": "^2.2.1",
    "ts-jest": "25.3.1",
    "ts-loader": "^6.0.4",
    "typescript": "^3.7.5",
    "wait-on": "^3.3.0",
    "webpack": "^4.35.3",
    "webpack-cli": "^3.3.6",
    "webpack-node-externals": "^1.7.2"
  },`

**Comment by djfdyuruiry** [comment]
@felixgourdeau and others in this thread - I have just published v2.1.0 which improves the handling of large `package.json` files which generate a huge amount of audit JSON output. There should be no limit now to the amount of packages that can be processed.
